ui/src/main/webapp/components/ControlWrapper.stories.ts

```diff
@@ -0,0 +1,41 @@
+import type { Meta, StoryObj } from '@storybook/react';
+import ControlWrapper from './ControlWrapper';
+
+const meta = {
+    component: ControlWrapper,
+    title: 'Components/ControlWrapper',
+} satisfies Meta<typeof ControlWrapper>;
+
+export default meta;
+type Story = StoryObj<typeof meta>;
+
+const props = {
```

Comment:
we gets loose typing declaring props that way. Please declare it inline or set type for the object. Maybe like Story['args']